api/managementpb/ia/rules.proto

```diff
@@ -114,24 +114,26 @@ message ListAlertRulesResponse {
 }
 
 message CreateAlertRuleRequest {
-  // Template name.
+  // Template name. Can't be specified simultaneously with source_rule_id.
   string template_name = 1;
+  // ID of the rule that will be used as source. Can't be specified simultaneously with template_name.
+  string source_rule_id = 2;
```

Comment:
is it necessary to update numbers?

Reply:
No, but IMHO better to keep them ordered until it's beta.